eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -126,7 +127,7 @@ stages:
               - deployment: UploadSymbols
                 displayName: Upload Symbols to Symbols Server
                 condition: and(succeeded(), ne(variables['Skip.SymbolsUpload'], 'true'))
-                environment: nuget
+                environment: ${{parameters.PackageEnvironment}}
```

Comment:
I think we should update all the environment's to match this so none of the release steps partially pass and will eliminate the need for multiple approvals in the release.

Reply:
agreed.  updated.